Sources/RemoteNotification.swift

```diff
@@ -88,10 +126,68 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
         /// The localization arguments for the title shown on the Apple Watch. Derived from userInfo["aps"]["alert"]["title-loc-args"]
         public let wearableTitleLocalizationArguments: [String]?
         
-        init?(remoteNotification: [String : AnyObject]) {
+        // MARK: Equatable
+        
+        public static func ==(lhs: Alert, rhs: Alert) -> Bool {
```

Comment:
Entire contents of method were pulled from below. Swift 3 allows inline `==` methods.